src/com/xilinx/rapidwright/util/RunVivado.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021-2022, Xilinx, Inc.
```

Comment:
Don't think this needs Xilinx copyright anymore. For new files, there's a `make ensure_headers` target that should generate the copyright header for you. Can you try that please?